firmware/hdmi_in0.c

```diff
@@ -19,8 +19,6 @@
 int hdmi_in0_debug;
```

Comment:
Should the `firmware/hdmi_in0.c` file be generated via the Makefile in some way now?

Reply:
I don't see why?